code/src/reusable/url.js

```diff
@@ -0,0 +1,5 @@
+
+// GET This will return the latest 20 thoughts from the API
+export const API_URL = "https://happy-thoughts-technigo.herokuapp.com/thoughts"
+
+export const API_URL_LIKES = (messageID) => `https://happy-thoughts-technigo.herokuapp.com/thoughts/${messageID}/like` //why do I need backticks and $, no ""?, --to pass argument to a string.
```

Comment:
Not sure if I understand the question but what comes to my mind is that we are using the template literals to pass a value from API and they need the back ticks to be recognized. But again, the coaches will explain better. :)